test/scenes/LavaScene.ts

```diff
@@ -13,7 +13,11 @@ export class LavaScene extends Phaser.Scene {
         });
     }
 
-    preload(): void {}
+    preload(): void {
+        // TODO instead of loading images to elapse time for scene create event to be added,
```

Comment:
See if there's a way to delay the scene creation in a different way, so the scene creation event in `PhaserTest.setup` can still be added consistently.

Reply:
To continue work in #51.